src/main.rs

```diff
@@ -218,7 +218,7 @@ fn main() {
             debug!("Writing timestamp file in: {:?}", path);
             match Timestamp::new().store(path.as_path()) {
                 Ok(_) => {}
-                Err(e) => error!("Failed to write timestamp file: {}", e),
+                Err(e) => error!("Failed to write timestamp file: {:?}", e),
```

Comment:
why did this change?

Reply:
anyhow has, to my knowledge, 2 different error output options. One just prints a simple one-line error message, which is usually just the error context. The other is more detailed and prints the entire error chain. Here is an example of the second option, taken from anyhow's docs: 
```
Error: Failed to read instrs from ./path/to/instrs.json

Caused by:
    No such file or directory (os error 2)
```

I opted for the second option as it is more detailed and more helpful if one runs into an error.

Reply:
ah perfect - for some reason I thought it was `{:#}` but the docs say it's `{:?}` like you have here: https://docs.rs/anyhow/latest/anyhow/struct.Error.html#display-representations